test/verify/check-login

```diff
@@ -364,6 +364,60 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
                                     "couldn't parse login input: Malformed input",
                                     "couldn't parse login input: Authentication failed")
 
+    @skipImage("No SELinux", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1804", "ubuntu-2004")
+    @skipImage("No semanage", "fedora-coreos")
+    def testSELinuxRestrictedUserSudo(self):
+        m = self.machine
+        b = self.browser
+
+        # Configure only sudo, and without using cockpit-askpass
+        m.write("/usr/share/cockpit/shell/override.json", """
+{ "bridges": [
+        {
+            "match": {
+                "superuser": null
+            },
+            "spawn": [
+                "/usr/bin/sudo",
+                "cockpit-bridge",
+                "--privileged"
+            ],
+            "privileged": true
+        }
+
+] }
+""")
+        self.addCleanup(m.execute, "rm /usr/share/cockpit/shell/override.json")
+
+        # No password so that it all works without cockpit-askpass
+        m.write("/etc/sudoers.d/admin", "admin ALL=(ALL) NOPASSWD:ALL")
+        self.addCleanup(m.execute, "rm /etc/sudoers.d/admin")
+
+        # Verify that this setup works as expected
+        m.start_cockpit()
+        b.login_and_go("/playground/test")
+        b.click(".super-channel button")
+        b.wait_in_text(".super-channel span", 'result: ')
+        self.assertIn('result: uid=0', b.text(".super-channel span"))
+        b.logout()
+
+        # Now as sysadm_u
+        m.execute("semanage login -a -s sysadm_u admin")
+        self.addCleanup(m.execute, "semanage login -d -s sysadm_u admin")
+
+        m.start_cockpit()
+        b.login_and_go("/playground/test")
+        b.click(".super-channel button")
+        b.wait_in_text(".super-channel span", 'result: ')
+
+        # This should work, but it will not because of
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1814569
+        working_images = [ ] # none yet :-(
```

Comment:
At some point we want to turn this into a list of broken_images, but for now that's fine -- if it starts to work on any image, the test will fail, which is all that we need.